frontend/src/hooks.ts

```diff
@@ -419,3 +419,9 @@ export function useIsMetaKeyDown(): boolean {
 
   return isDown;
 }
+
+export function useTitle(title: string): void {
+  useEffect(() => {
+    document.title = `Samfundet | ${title}`;
+  }, [title]);
+}
```

Comment:
Ser bra ut! Men jeg tror det trengs en måte å sette tittel uten ` | `. For eksempel om man navigerer til Informasjon > Generelt, så blir tittelen `Samfundet | Om Samfundet`, og hvis man går tilbake til hjemmesiden vil man forvente at den blir kun `Samfundet`, men det blir den ikke, og det er ingen måte nå å gjøre det. 

Tror også det kan være greit å ha støtte for å ha andre navn enn Samfundet, for eksempel Lyche-sider trenger ikke å ha Samfundet i tittelen i det hele tatt.

Flere måter man kan løse det på, men under er et forslag. Jeg foretrekker også å ha "Samfundet" bak navnet på siden. Om man har mange tabs åpne vil det være enklere å se hva som er hva.

```suggestion
export function useTitle(title: string, suffix: string = 'Samfundet'): void {
  useEffect(() => {
    document.title = title ? `${title} | ${suffix}` : suffix;
  }, [title, suffix]);
}
```

Om du går for denne løsningen vil `useTitle('')` gi `Samfundet` som tittel, og `useTitle('Reservasjon', 'Lyche')` gi `Reservasjon | Lyche`

Reply:
@robines bare gjøre disse endringene, så få den inn?